src/Nest/XPack/MachineLearning/PostJobData/PostJobDataResponse.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace Nest
+{
+	public interface IPostJobDataResponse : IResponse
+	{
+		/// <summary>
+		/// The unique identifier for the job
+		/// </summary>
+		[JsonProperty("job_id")]
+		string JobId { get; }
+
+		[JsonProperty("processed_record_count")]
```

Comment:
This PR is full of awesome xmldocs but this one is left out 😿

Reply:
Done.